app/src/main/java/de/codefor/karlsruhe/opensense/widget/plot/PlotWidget.kt

```diff
@@ -41,6 +36,11 @@ class PlotWidget : BaseWidget() {
     }
 
     companion object {
+        private val dateTimeFormatter = DateTimeFormat.forPattern("HH:mm")
+        // Use a different formatter without full the date for the first and last tick
+        private val dateTimeFormatterEdges = DateTimeFormat.forPattern("d. M., HH:mm")
```

Comment:
these format strings are accidentally reversed, the edge formatter should be shorter than the non-edge formatter

Reply:
Oh sorry, that was my mistake while refactoring the code. It's fixed with the last commit.